modules/silicon_design/scripts/build/notebooks/cfuplayground/cfuplayground.md

```diff
@@ -0,0 +1,74 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# CFU Playground
+
+
+### Design Space Exploration
+
+```python tags=["parameters"]
+# Vexriscv soft core parameters available for tuning
+bypass            = True
+cfu               = True
+dCacheSize        = 2048
+hardwareDiv       = True
+iCacheSize        = 1024
+mulDiv            = True
+prediction        = "none"
+safe              = True
+singleCycleShift  = True
+singleCycleMulDiv = True
+```
+
+```python
+# Constants 
+CSR_PLUGIN_CONFIG = "mcycle"
+TARGET            = "digilent_arty"
+```
+
+```python
+# Change directory to design space exploration project in CFU-Playground
+%cd /CFU-Playground/proj/dse_template
+```
+
+```python tags=[]
+import dse_framework
+
+dse_framework.dse(CSR_PLUGIN_CONFIG, bypass, cfu, dCacheSize, hardwareDiv, iCacheSize, mulDiv, prediction, safe, singleCycleShift, singleCycleMulDiv)
```

Comment:
is the `dse_framework` already in CFU Playground?

Reply:
it is in the process of getting merged but is on my branch which is what I am cloning in the provision script for now so that i can test this PR in parallel

Reply:
then maybe you could update the provisionning script to fetch  your branch instead?

Reply:
it already does that